code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -598,7 +598,13 @@
 		delay = 1 SECONDS
 		playsound(loc, "alien_doorpry", 25, TRUE)
 	else
-		delay = 4 SECONDS
+		switch(M.mob_size)
+			if(MOB_SIZE_XENO_SMALL, MOB_SIZE_XENO_VERY_SMALL)
+				delay = 4 SECONDS
+			if(MOB_SIZE_BIG)
```

Comment:
i dont know if mob size is the right thing . defenders will pry doors super fast